lib/AST/ASTContext.cpp

```diff
@@ -1058,19 +1058,11 @@ ASTContext::getBuiltinInitDecl(NominalTypeDecl *decl,
   auto builtinProtocol = getProtocol(builtinProtocolKind);
   auto builtinConformance = getStdlibModule()->lookupConformance(
       type, builtinProtocol);
-  if (builtinConformance.isInvalid()) {
-    assert(false && "Missing required conformance");
-    witness = ConcreteDeclRef();
-    return witness;
-  }
+  assert(!builtinConformance.isInvalid && "Missing required conformance");
```

Comment:
I think you forgot the parens:

```suggestion
  assert(!builtinConformance.isInvalid() && "Missing required conformance");
```